assignment2/assignment2/Presentation/Login/KakaoLoginViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  KakaoLoginViewController.swift
+//  assignment2
+//
+//  Created by 이승헌 on 2022/10/12.
+//
+
+import UIKit
+import SnapKit
+
+protocol KakaoLoginViewDelegate: AnyObject {
```

Comment:
여기서 AnyObject 로 하는 것도 좋지만, 만약 채택할 대상이 정해졌다면 명시적으로 정해주는 방법도 좋아요! (다른 곳에서는 사용할 수 없도록 하는 것이니까!!)
예를 들어서 이 프로토콜을 KakaoLoginViewController에서 채택해서 사용한다면?
```suggestion
protocol KakaoLoginViewDelegate: KakaoLoginViewController {
```
이렇게 하는 것이죠!! 

이게 답이다!! 라는 것은 아니고 .. 한번 참고해보시면 좋을 것 같아용가릐